add-changelog-100124

tirumerla opened the pull request:
- Add changelog for jan 10th release

ggonzalez94 commented on docs/modules/ROOT/pages/changelog.adoc:
```diff
@@ -1,5 +1,43 @@
+** Fix deployment status when deploy fails in the backend.
+
+* *Code Inspector*
+** Feature to allow users to specify settings and configuration of a repository via a file called "defender.config.json" on the root directory.
```
I don't think we are including this as part of the release are we? I see the [ticket](https://linear.app/openzeppelin-development/issue/PLAT-3073/report-configuration-via-file-on-repository) is still showing as merged but never made it to staging.

tirumerla replied:
Good catch, we need to update release drafter looks like the reverts have been updated in drafter.

zanbel reviewed (changes requested):
See my suggested changes below. I mainly removed content that doesn't provide any value to users like "minor bug fix".

I do have a few questions
- What does Private and Tenant networks mean? Is it meant to be Private and Forked networks? If the answer is yes, we should split them and consider each as a separate feature even if the underline implementation is very similar 
- "Allow role-base access control changes by auditors" means we added a new role?
- "Access to tenant networks with read-only permissions." is this also a role based access control role added?
- "Apply restrictions on the usage of specific chain ids." which restrictions?

---
* *General*
** Minor bug fixes and security improvements across all components.

* *Authorization*
** Allow team members to re-send an invitation for pending users.

* *Monitor*
** Forta monitors added support for Optimism and Fantom networks.

* *Private & Tenant Networks*
** Access to tenant networks with read-only permissions.
** Apply restrictions on the usage of specific chain ids.

* *Audit*
** Audit report logo and code block spacing styling improvements.
** Allow role-base access control changes by auditors.

* *Deploy*
** Fix deployment status when the deploy request fails.

* *Code Inspector*
** Allow users to directly request an audit if there are reports with more than 10 findings.

tirumerla commented:
Thanks for the review. See the responses

> I do have a few questions
> 
> * What does Private and Tenant networks mean? Is it meant to be Private and Forked networks? If the answer is yes, we should split them and consider each as a separate feature even if the underline implementation is very similar

Thought we changed the name from forked networks to tenant networks, may be i'm mistaken here. I see some have tenant networks name and some forked networks and some private networks 

> * "Allow role-base access control changes by auditors" means we added a new role?

we are allowing auditors to make changes to role permissions of other auditors
[PLAT-3334 Cannot modify OZ user permissions](https://linear.app/openzeppelin-development/issue/PLAT-3334/cannot-modify-oz-user-permissions)

> * "Access to tenant networks with read-only permissions." is this also a role based access control role added?
[PLAT-2955 User with forked Networks read only permission is not able to view Forked Network details](https://linear.app/openzeppelin-development/issue/PLAT-2955/user-with-forked-networks-read-only-permission-is-not-able-to-view)


> * "Apply restrictions on the usage of specific chain ids." which restrictions?

Forked Networks — continue to restrict so that cannot reuse a public chain ID

Private Networks — only restrict so that cannot reuse a chain ID that is officially supported by Defender

[PLAT-3654 Narrow restriction on Chain IDs](https://linear.app/openzeppelin-development/issue/PLAT-3654/narrow-restriction-on-chain-ids)
